src/vctrs.h

```diff
@@ -383,6 +382,11 @@ SEXP vec_group_loc(SEXP x);
 SEXP vec_match_params(SEXP needles, SEXP haystack, bool na_equal,
                       struct vctrs_arg* needles_arg, struct vctrs_arg* haystack_arg);
 
+#include "cast.h"
+static inline SEXP vec_cast(SEXP x, SEXP to, struct vctrs_arg* x_arg, struct vctrs_arg* to_arg) {
```

Comment:
Is it worth it to move this and `vec_cast_common` and maybe `vec_cast_e` to `cast.h`?

Reply:
Yes, I think we should clean the header files in one go after the release.